modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/group/src/raylife-ap-react-fragments/fragments/applications-status/index.js

```diff
@@ -29,93 +29,91 @@ export default function () {
-			getApplicationsStatus(CONSTANTS.STATUS.REJECTED),
-			getApplicationsStatus(CONSTANTS.STATUS.REVIEWED),
-			getApplicationsStatus(CONSTANTS.STATUS.UNDERWRITING),
-		]).then((results) => {
-			const [
-				boundApplicationsResult,
-				incompleteApplicationsResult,
-				quotedApplicationsResult,
-				openApplicationsResults,
-				rejectedApplicationsResult,
-				reviewedApplicationsResult,
-				underwritingApplicationsResult,
-			] = results;
-
-			const colors = {
-				bound: '#D9E4FE',
-				incomplete: '#1F77D4',
-				open: '#FF7F0E',
-				quoted: '#81A8FF',
-				rejected: '#191970',
-				reviewed: '#4C84FF',
-				underwriting: '#B5CDFE',
-			};
-
-			const cols = [
-				[
-					CONSTANTS.STATUS.OPEN,
-					getTotalCount(openApplicationsResults),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.OPEN),
-				],
-				[
-					CONSTANTS.STATUS.INCOMPLETE,
-					getTotalCount(incompleteApplicationsResult),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.INCOMPLETE),
-				],
-				[
-					CONSTANTS.STATUS.QUOTED,
-					getTotalCount(quotedApplicationsResult),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.QUOTED),
-				],
-				[
-					CONSTANTS.STATUS.UNDERWRITING,
-					getTotalCount(underwritingApplicationsResult),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.UNDERWRITING),
-				],
-				[
-					CONSTANTS.STATUS.REVIEWED,
-					getTotalCount(reviewedApplicationsResult),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.REVIEWED),
-				],
-				[
-					CONSTANTS.STATUS.REJECTED,
-					getTotalCount(rejectedApplicationsResult),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.REJECTED),
-				],
-				[
-					CONSTANTS.STATUS.BOUND,
-					getTotalCount(boundApplicationsResult),
-					setFirstLetterUpperCase(CONSTANTS.STATUS.BOUND),
-				],
-			];
-
-			const columns = cols.filter((col) => col[1] > 0);
-
-			setChartData({...chartData, ...{colors, columns}});
-
-			const title = columns
-				.map((array) => array[1])
-				.reduce((sum, i) => {
-					return sum + i;
-				})
-				.toString();
-
-			setChartTitle(title);
-
-			setLoadData(true);
-		});
+	const loadChartData = async () => {
+		const colors = {
+			bound: '#D9E4FE',
+			incomplete: '#1F77D4',
+			open: '#FF7F0E',
+			quoted: '#81A8FF',
+			rejected: '#191970',
+			reviewed: '#4C84FF',
+			underwriting: '#B5CDFE',
+		};
+
+		const openStatus = await getApplicationsStatus(CONSTANTS.STATUS.OPEN);
+
+		const incompleteStatus = await getApplicationsStatus(
+			CONSTANTS.STATUS.INCOMPLETE
+		);
+
+		const quotedStatus = await getApplicationsStatus(
+			CONSTANTS.STATUS.QUOTED
+		);
+
+		const underwritingStatus = await getApplicationsStatus(
+			CONSTANTS.STATUS.UNDERWRITING
+		);
+
+		const reviewedStatus = await getApplicationsStatus(
+			CONSTANTS.STATUS.REVIEWED
+		);
+
+		const rejectedStatus = await getApplicationsStatus(
+			CONSTANTS.STATUS.REJECTED
+		);
+
+		const boundStatus = await getApplicationsStatus(CONSTANTS.STATUS.BOUND);
+
+		const cols = [
```

Comment:
What does "cols" mean? Maybe you should try a more clear name.

Reply:
I do not know. :stuck_out_tongue: